src/axom/inlet/Inlet.hpp

```diff
@@ -418,12 +419,28 @@ class Inlet : public SchemaCreator
    * \return Shared pointer to the created Field
    *****************************************************************************
    */
-  std::shared_ptr<Table> addStringArray(const std::string& name,
-                                        const std::string& description = "")
+  std::shared_ptr<Verifiable> addStringArray(const std::string& name,
+                                             const std::string& description = "")
   {
     return m_globalTable->addStringArray(name, description);
   }
 
+  /*!
+   *****************************************************************************
+   * \brief Add an array of Fields to the input deck schema.
+   *
+   * \param [in] name Name of the array
+   * \param [in] description Description of the Field
+   *
+   * \return Shared pointer to the created Field
+   *****************************************************************************
+   */
+  std::shared_ptr<Table> addGenericArray(const std::string& name,
```

Comment:
Why is this `Table` not `Verifiable` like the rest of the `add*Array` functions?

Reply:
Arrays of structs within arrays of structs are not yet supported.  I was hoping to do that in a future PR as it would require a new interface that (probably) extends `Verifiable` but also supports adding new fields.